### PR TITLE
Add contact email as an optional profile property

### PR DESCRIPTION
**Pull request by rushtong**
Addresses https://broadinstitute.atlassian.net/browse/GAWB-748
This PR has a companion in ui: https://github.com/broadinstitute/firecloud-ui/pull/449
- [x] **Submitter**: Rebase to develop. DO NOT SQUASH
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: Make sure documentation for code is complete
- [x] **Submitter**: Review code comments; remove done TODOs, create stories for remaining TODOs
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Add description or comments on the PR explaining the hows/whys (if not obvious)
- [x] **Submitter**: Tell tech lead that the PR exists if s/he wants to look at it
- [x] **Submitter**: Anoint a lead reviewer (LR). **Assign PR to LR**
- [x] **LR**: Initial review by LR and others.
- [x] Comment / review / update cycle:
  - Rest of team may comments on PR at will
  - **LR assigns to submitter** for feedback fixes
  - Submitter updates documentation as needed
  - Submitter rebases to develop again if necessary
  - Submitter makes further commits. DO NOT SQUASH. **Reassign to LR** for further feedback
- [x] **Tech lead**:  sign off
- [x] **LR**: sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Squash commits, rebase if necessary
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Merge to develop 
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: Check configuration files in Jenkins in case they need changes
- [x] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [x] **Submitter**: Verify swagger UI on dev server still works after deployment
- [x] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala [hunk 1]
```diff
@@ -67,7 +71,7 @@ case class Profile (
 object Profile {
 
   // increment this number every time you make a change to the user-provided profile fields
-  val currentVersion:Int = 2
+  val currentVersion:Int = 3
```
Comment (helgridly) [inline src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala:74]: i don't have a better solution and i'm not requesting you change it, but i'd like to register my extreme displeasure at this. it's horrifying

Comment (rushtong) [inline src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala:74]: Duly noted :-)

**Comment by helgridly** [comment]
:+1:

**Comment by dmohs** [comment]
👍  Looks good to me (as interim tech lead)

File: src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala (hunk 1 above)
Comment (davidangb) [inline src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala:74]: @helgridly can you explain your displeasure?

Comment (helgridly) [inline src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala:74]: It's a magic number that developers need to remember to increment manually when something else changes. I don't like remembering things and I don't like manual things. I'd prefer deriving a hash or something from the class definition itself.

Comment (helgridly) [inline src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala:74]: (But I don't prefer it enough to insist on it, which is why I didn't insist on it.)

**Comment by davidangb** [comment]
👍

**Pull request merged**